### ISSUE TITLE
[TableOfContents]: missing WCAG attributes

### ISSUE THREAD

**Issue by helemand** [TEHIK-EE/tedi-design-system#163]
### Description of the problem

1. For the nested table of contents JAWS does not say whether the item is open or not.
Requirement: [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)

**Comment by helemand** [comment]
solved in [#164](https://github.com/TEHIK-EE/tedi-design-system/pull/164)
